knora/dsplib/utils/excel_to_json_lists.py

```diff
@@ -44,20 +46,21 @@ def expand_lists_from_excel(
             new_lists.append(_list)
         else:
             # this is a reference to a folder with Excel files
-            prepared_rootnode, excel_file_names = _prepare_list_creation(
```

Comment:
Due to the new design of the `excel2lists` command, a number of simplifications could happen. One simplification is that it is no longer necessary to prepare a rootnode, so I renamed the method to `_extract_excel_file_pathes`.